packages/cli/config/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "incremental": true,
     "isolatedModules": true,
     "jsx": "react",
-    "lib": ["DOM", "DOM.Iterable", "ScriptHost", "ES2019", "ESNext.Promise"],
+    "lib": ["DOM", "DOM.Iterable", "ScriptHost", "ES2020", "ESNext.Promise"],
```

Comment:
@ryanvazquez What's the background for doing this?

Reply:
`Promise.allSettled` is a ES2020 feature used in this PR, not supported in ES2019

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled

Reply:
This also means that it isn't transpiled/translated, and that all platforms must support it, right?

Seems it was introduced in node 12.9, and the caniuse level is decent ... @Rugvip @benjdlambert any opinions on how high to go on the ES level?

Reply:
I think it's acceptable. It's supported in Chrome at least. And we're moving to node v14 soon too.

Reply:
I don't know if we have some form of support table for browsers yet, but lets aim for latest chrome 🙌

Reply:
> This also means that it isn't transpiled/translated, and that all platforms must support it, right?

Yep, which is why this is a pretty big ask. This PR isn't dependent on this feature and it can be reimplemented for our use case it but I'd prefer to use the native option where possible.

Reply:
I think `Promise.allSettled` is alright, but we'd have to go through all of the `ES2020` features and check compatibility. Iirc I selected `ES2019` because there wasn't broad support for all `ES2020` features yet

Reply:
Seems like only `string.matchAll` doesn't have 100% support from node yet, but it's about error checking behaviour being a bit off. Other missing features are `??` and `?.`, which are both transpiled for now. I'm alright with bumping up to `ES2020` 👍

Reply:
Thanks!